app/controllers/picks_controller.rb

```diff
@@ -19,7 +19,15 @@ def show
   end
 
   def create
-    @pick = Pick.new(pick_params)
+    @pick = Pick.create(pick_params)
+    respond_to do |format|
+      format.jsonapi { render jsonapi: @pick }
+    end
+  end
+
+  def destroy
+    @pick = Pick.find(params[:id])
+    @pick.destroy
```

Comment:
This looks good but maybe want to just return something to the sender below this?

```
    respond_to do |format|
      format.jsonapi { head :no_content }
    end
```

Also I think if you want to shorten it, you get `@pick` for free when you're throwing it the id of the pick. So the full block for offer is:

```
  def destroy
    @offer.destroy
    respond_to do |format|
      format.jsonapi { head :no_content }
    end
  end
```